src/bundle/Resources/views/content/tabs/versions.html.twig

```diff
@@ -0,0 +1,112 @@
+<div class="ez-versions-list-container">
+    {%  if draftVersions %}
+        <h2 class="ez-list-toolbar-label"><th>{{'locationview.versions.draft.under.edit'|trans({}, 'locationview')}}</th></h2>
+        <table class="ez-table-data">
+            <thead>
+            <tr>
+                <th>{{'locationview.versions.version'|trans({}, 'locationview')}}</th>
+                <th>{{'locationview.versions.modified.language'|trans({}, 'locationview')}}</th>
+                <th>{{'locationview.versions.author'|trans({}, 'locationview')}}</th>
+                <th>{{'locationview.versions.created'|trans({}, 'locationview')}}</th>
+                <th>{{'locationview.versions.last.saved'|trans({}, 'locationview')}}</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for version in draftVersions %}
```

Comment:
this could be moved into an include, seems all 3 version lists use the same html?